src/com/layouts/sidebar/sidebar.js

```diff
@@ -0,0 +1,29 @@
+import {h, Component} from 'preact/preact';
+
+import ViewBar 							from 'com/view-bar/bar';
+import ViewHeader						from 'com/view-header/header';
+import ViewSidebar						from 'com/view-sidebar/sidebar';
+import ViewContent						from 'com/view-content/content';
+import ViewFooter						from 'com/view-footer/footer';
+
+
+export default class LayoutSidebar extends Component {
```

Comment:
`LayoutWithSidebar` would be clearer.

Also this folder should be `layout`, and the file `layout/with-sidebar/with-sidebar.js`.

Reply:
I called it LayoutSidebar because we might want layouts without the sidebar. That's actually why I removed the noSidebar prop since I didn't think I'd need it.

Reply:
The code contains unfinished sidebar support. I've included a one-line fix to add it.